documentation/messages/user.registration.transactional.md

```diff
@@ -0,0 +1,121 @@
+# User registration transactional message
+
+User registration event.
+
+**Routing key**
+
+`user.registration.transactional`
+
+**Exchanges**
+- `transactionalExchange`
+
+**Queues**
+- `activityStatsQueue`
+- `loggingQueue`
+- `mobileCommonsQueue`
+- `transactionalQueue`
+- `userAPIRegistrationQueue`
+- `userRegistrationQueue`
+
+**Parameters**
+
+```js
+{
+
+  /* Required. Always must be `user_register`. */
+  activity: String,
+
+  /* Required. */
+  email: String,
+
+  /* Required. */
+  uid: Number,
+
+  /* Required, Example: `US`. */
+  user_country: String,
+
+  /* Required, Example: `en`. */
+  user_language: String,
+
+  /* Required. Example: `mb-user-register-US`. */
+  email_template: String,
+
+  /* Required. Example: `f2fab1dfd4`. */
+  mailchimp_list_id: String,
```

Comment:
Optional -  if user `user_country` is defined and the `activity: user_register` then this can have a default value.